integrations/symfony-bundle.rst

```diff
@@ -205,6 +205,7 @@ services are:
 * ``httplug.factory.guzzle6``
 * ``httplug.factory.react``
 * ``httplug.factory.socket``
+* ``httplug.factory.mock`` (Install ``php-http/mock-client``first)
```

Comment:
i think you need a whitespace before the "first)" bit to make sphinx build this correctly